src/dataDisplay/popover/__test__/popover.test.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * To run this test from the church-management/client folder, execute the following command:
```

Comment:
Can we tweak this slightly for appropriateness in React CM UI?
I'm thinking just: `To run this test, execute the following command:`
Since it's from the root of the project.

That having been said, this comment apparently exists in many unit test fixtures in React CM UI, and none of us noticed until now!  So, hardly a blocker.